topydo/lib/PrettyPrinterFilter.py

```diff
@@ -32,13 +32,16 @@ def filter(self, p_todo_str, _):
         return p_todo_str
 
 PRIORITY_COLORS = {
-    'A': '\033[36m', # cyan
-    'B': '\033[33m', # yellow
-    'C': '\033[34m'  # blue
+    'A': '\033[0;36m', # cyan
+    'B': '\033[0;33m', # yellow
+    'C': '\033[0;34m'  # blue
 }
 
-PROJECT_COLOR = '\033[31m' # red
-NEUTRAL_COLOR = '\033[0m'
+PROJECT_COLOR  = '\033[1;31m' # color for + keyword  : red
+CONTEXT_COLOR  = '\033[1;35m' # color for @ keyword  : magenta
+METADATA_COLOR = '\033[1;32m' # color for @ metadata : green
```

Comment:
Drop ' @' from ' @ metadata' in comment as metadata doesn't start with '@'. Same for '@ link'